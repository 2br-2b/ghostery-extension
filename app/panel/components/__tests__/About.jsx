import React from 'react';
import renderer from 'react-test-renderer';
import About from '../About';


// Importing Mock Data
jest.mock('../../../../src/classes/Globals', () => ({
	BROWSER_INFO: { displayName: 'Chrome' },
	EXTENSION_VERSION: '8.3.2'
}));

describe('app/panel/components/About', () => {
	describe('Snapshot tests with react-test-renderer', () => {
		test('About component is rendered correctly', () => {
			const component = renderer.create(
				<About />
			).toJSON();
			expect(component).toMatchSnapshot();
		});
	});
});

