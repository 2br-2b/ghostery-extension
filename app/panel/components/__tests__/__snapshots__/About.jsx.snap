// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/panel/components/About Snapshot tests with react-test-renderer About component is rendered correctly 1`] = `
<div
  id="content-about"
>
  <div
    className="row"
  >
    <div
      className="small-12 columns"
    >
      <h1>
        panel_about_panel_header
      </h1>
      <div
        className="support-section"
      >
        <h3>
          panel_about_version_header
        </h3>
        <a
          href="https://github.com/ghostery/ghostery-extension/releases"
          rel="noopener noreferrer"
          target="_blank"
        >
          panel_about_release_notes
        </a>
        <a
          href="https://www.mozilla.org/en-US/MPL/2.0/"
          rel="noopener noreferrer"
          target="_blank"
        >
          panel_about_license
        </a>
        <a
          href="https://www.ghostery.com/about-ghostery/browser-extension-privacy-policy/"
          rel="noopener noreferrer"
          target="_blank"
        >
          panel_about_privacy_statement
        </a>
        <a
          href="https://www.ghostery.com/about-ghostery/ghostery-terms-and-conditions/"
          rel="noopener noreferrer"
          target="_blank"
        >
          panel_about_terms_and_conditions
        </a>
        <a
          href="https://www.ghostery.com/about-ghostery/imprint/"
          rel="noopener noreferrer"
          target="_blank"
        >
          panel_about_imprint
        </a>
        <div
          onClick={[Function]}
        >
          panel_about_licenses
        </div>
        <a
          href="https://www.ghostery.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Ghostery.com
        </a>
      </div>
    </div>
  </div>
</div>
`;
